pkgs/by-name/so/sold/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sold";
+  version = "0-unstable-2024-02-22";
+
+  src = fetchFromGitHub {
+    owner = "bluewhalesystems";
+    repo = pname;
+    rev = "a52e8b37661562c866a579ab1164f94694f479a5";
+    sha256 = "0iyrgz0ay2453rqrlvxsrla0998l8yq6q3xyqqw5icfvh05ld8i8";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  enableParallelBuilding = true;
+
```

Comment:
openssl seems to be required at build time and run time (at least on x86_64-linux)
so

```nix
  buildInputs = [ openssl ];
```

Reply:
Done, thank you!